docs/code-snippets/outlook-snippets.yaml

```diff
@@ -1359,8 +1367,8 @@ Office.Recurrence.setAsync:
   - |-
     // The following example sets the recurrence pattern of an appointment series.
     var seriesTimeObject = new Office.SeriesTime(); 
-    seriesTimeObject.setStartDate(2017,10,2);  
```

Comment:
What happens if the start and date are in the past? Maybe we should just make these dates span a large interval (2017-2030)?

Reply:
It would work just fine. I actually tried it with these past dates and there was no issue.